napari/components/viewer_model.py

```diff
@@ -198,7 +198,6 @@ def __init__(self, title='napari', ndisplay=2, order=(), axis_labels=()):
         self.dims.events.order.connect(self._update_layers)
         self.dims.events.order.connect(self.reset_view)
         self.dims.events.current_step.connect(self._update_layers)
-        self.cursor.events.position.connect(self._on_cursor_position_change)
         self.cursor.events.position.connect(
             throttled(self._update_status_bar_from_cursor, timeout=50)
         )
```

Comment:
If we left this here, there should be a proper update of documentation of `status` event to warn people that this event may be emitted outside the main thread.

Reply:
Sounds good, but I'm not sure how/where to document it. I guess in the main docstring of `ViewerModel`? Though that is currently missing a number of fields including `status`.

I think there should be a follow-up PR to this one that does move throttling into the qt part of the code, but like I said I'm unsure how to do that right now. Happy to create an issue to track that work.